services/transport/https/graphql/resolver-creator.go

```diff
@@ -177,7 +179,23 @@ func (creator *resolverCreator) NewEdgeClusterTypeEdgeResolver(
 // ctx: Mandatory. Reference to the context
 // Returns the EdgeClusterTypeConnectionResolverContract or error if something goes wrong
 func (creator *resolverCreator) NewEdgeClusterTypeConnectionResolver(ctx context.Context) (edgecluster.EdgeClusterTypeConnectionResolverContract, error) {
-	return queryedgecluster.NewEdgeClusterTypeConnectionResolver(ctx, creator)
+	return queryedgecluster.NewEdgeClusterTypeConnectionResolver(
+		ctx,
```

Comment:
deliberately changed ?

Reply:
Of course, code formatting, and also making functions with more than one parameters to be broken into multiple line